### ISSUE TITLE
[InputForm.json]: While navigating the Input form screen after entering incorrect mail, User is not provided the correct format for a valid email address.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#107]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Open Adaptive Cards Visualizer Application.
2. Navigate to sample cards using tab key and activate it using Enter key.
3. A dropdown would appear. Activate it using Enter key.
4. Navigate to InputForms using tab key and activate it using Enter key.
5. Navigate to the Visual tab item present at the top. A sample card would appear.
6. Navigate to the email edit box inside the sample card using tab key.
7. Observe the issue.

**Actual Result:**
User is not provided the correct format for a valid email address. It just displays "Enter a valid email address" after the user has entered in another format.

**Expected Result:**
User should be provided the correct format for a valid email address. It should be more descriptive and display something as "Enter a valid email address i.e abc@xyz.com".

**User Experience:**
All users would get affected if User is not provided the correct format for a valid email address.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

![User is not provided the correct format for a valid email address](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/efe3b2c3-b217-4cea-85f4-c8838bf83fc7)

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/4f8a653e-9278-4ca9-844b-814b1655eee8

### Card JSON

```JSON
NA
```
